Valid annotation is missing from autogenerated files

boat plugin 0.17.51 is missing the import of the @Valid annotation (import jakarta.validation.Valid )

error: cannot find symbol
[ERROR]   symbol:   class Valid

--- a/boat-scaffold/src/main/templates/boat-java/libraries/webclient/api.mustache
+++ b/boat-scaffold/src/main/templates/boat-java/libraries/webclient/api.mustache
@@ -6,6 +6,11 @@ import {{invokerPackage}}.ApiClient;
 {{/imports}}
 {{^fullJavaUtil}}
 
+{{#useBeanValidation}}
+import jakarta.validation.constraints.*;
+import jakarta.validation.Valid;
+{{/useBeanValidation}}
+
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;

